### PR TITLE
Have automatic tagging as an option

### PR DESCRIPTION
**Pull request by ychartois**
Not everybody would like to have automatic tagging as it's tied to personal workflow. It perfect when your workflow follow the: 1 task = 1 PR, but it's not always the case.
By default the automatic tagging is turned off, if you want to enable it, you have to add it to your `teamwork.yml` like:

```
        with:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
          TEAMWORK_URI: ${{ secrets.TEAMWORK_URI }}
          TEAMWORK_API_TOKEN: ${{ secrets.TEAMWORK_API_TOKEN }}
          AUTOMATIC_TAGGING: false

```

Fixes https://github.com/Teamwork/github-sync/pull/33

### CODE REVIEW

File: src/main.sh [hunk 1]
```diff
@@ -12,11 +12,11 @@ main() {
   # Ensure env vars and args exist
   ensure::env_variable_exist "GITHUB_REPOSITORY"
   ensure::env_variable_exist "GITHUB_EVENT_PATH"
-  ensure::total_args 3 "$@"
```
Comment (toffentoffen) [inline src/main.sh:15]: What do you think about checking if `TEAMWORK_API_TOKEN` exists as well? Isn't it mandatory as well?

**Review by toffentoffen — COMMENTED** [review]
Small doubt to discuss. In general LGTM

File: src/main.sh (hunk 1 above)
Comment (miguelbemartin) [reply src/main.sh:15]: I think we don't need to check it because it's already defined as mandatory in the YML file.

**Review by miguelbemartin — APPROVED** [review]
Code looks good to me. I didn't test locally.

**Comment by ychartois** [comment]
I did test but might have miss something... anyway, the "Ensure this projects runs" is stuck since yesterday...

**Comment by miguelbemartin** [comment]
> I did test but might have miss something... anyway, the "Ensure this projects runs" is stuck since yesterday...

This is because this only runs when:
```
on:
  pull_request:
    types: [opened, closed]
  pull_request_review:
    types: [submitted, dismissed]
```

so if you update the PR, the workflow to test the action doesn't run again.

**Comment by miguelbemartin** [comment]
@ychartois feel free to merge this PR. 

Don't forget to add a "new release" after PR has been merged.

**Pull request merged**